cvat-ui/src/components/quality-control/task-quality/allocation-table.tsx

```diff
@@ -37,10 +37,12 @@ interface RowData {
     active: boolean;
 }
 
+const COLUMNS_OFFSET_PERCENT = 0.30;
```

Comment:
PERCENT is something between 0 and 100. So, it should be probably 30 (if you do not mean here 0.3% of course).